CHANGELOG.md

```diff
@@ -1,5 +1,232 @@
 # Changelog
 
+## [1.0.0-Release Candidate 1](https://github.com/ably/ably-asset-tracking-swift/tree/v1.0.0-rc.1)
+[Full Changelog](https://github.com/ably/ably-asset-tracking-swift/compare/v1.0.0-beta.1...v1.0.0-rc.1)
+
+**Implemented enhancements:**
+
+- Enable setting a constant location engine resolution [\#276](https://github.com/ably/ably-asset-tracking-swift/issues/276)
+- Add ability to observe raw locations change in `Subscriber` SDK [\#272](https://github.com/ably/ably-asset-tracking-swift/issues/272)
+- Send batches of location updates [\#265](https://github.com/ably/ably-asset-tracking-swift/issues/265)
+- Upgrade Mapbox dependencies [\#260](https://github.com/ably/ably-asset-tracking-swift/issues/260)
+- Display resolution information in the subscriber example app [\#254](https://github.com/ably/ably-asset-tracking-swift/issues/254)
+- Send resolution to subscriber [\#253](https://github.com/ably/ably-asset-tracking-swift/issues/253)
+- Add switch enabling raw locations publishing in the publisher example app [\#252](https://github.com/ably/ably-asset-tracking-swift/issues/252)
+- Add raw locations to the Subscriber SDK [\#251](https://github.com/ably/ably-asset-tracking-swift/issues/251)
+- Send raw locations from the Publisher [\#250](https://github.com/ably/ably-asset-tracking-swift/issues/250)
+- Route is not recalculated when user is off route [\#184](https://github.com/ably/ably-asset-tracking-swift/issues/184)
+- Implement `RSC7d` \(Ably-Agent header\) [\#172](https://github.com/ably/ably-asset-tracking-swift/issues/172)
+- Implement stopping strategy [\#165](https://github.com/ably/ably-asset-tracking-swift/issues/165)
+- Test issue ignore [\#163](https://github.com/ably/ably-asset-tracking-swift/issues/163)
+- Send skipped location updates [\#135](https://github.com/ably/ably-asset-tracking-swift/issues/135)
+- Add token based authentication [\#134](https://github.com/ably/ably-asset-tracking-swift/issues/134)
+- Modify Publisher example app to work in background [\#110](https://github.com/ably/ably-asset-tracking-swift/issues/110)
+- Enable setting a constant location engine resolution [\#277](https://github.com/ably/ably-asset-tracking-swift/pull/277) ([lukasz-szyszkowski](https://github.com/lukasz-szyszkowski))
```

Comment:
Since this is a PR shouldn't it be under `Merged pull requests`? :thinking:

Reply:
Yeah, it is duplicated in a couple of places. I'll leave it only in the `Merged pull requests` section.